catroid/src/main/java/org/catrobat/catroid/transfers/project/ProjectUploadService.kt

```diff
@@ -128,6 +128,7 @@ class ProjectUploadService : IntentService("ProjectUploadService") {
                 ToastUtil.showError(this, resources.getString(R.string.error_project_upload) + " " + errorMessage)
                 StatusBarNotificationManager.getInstance()
                     .createUploadRejectedNotification(applicationContext, errorCode, errorMessage, reUploadBundle)
+                resultReceiver.send(0, null)
```

Comment:
was this the missing line?

Reply:
yes and in addition we now need strings as a result for the project id because share now uses uuid